auto_nag/config.py

```diff
@@ -19,15 +19,19 @@ def __init__(self):
         else:
             with open(MyConfig.PATH) as In:
                 self.conf = json.load(In)
-                if not self.conf.get("bz_api_key", None):
-                    raise Exception(
-                        "Your config.json file must contain a Bugzilla token"
-                    )
+
+        if "bz_api_key" not in self.conf:
+            raise Exception("Your config.json file must contain a Bugzilla token")
+
+        if "bz_api_key_nomail" not in self.conf:
+            self.conf["bz_api_key_nomail"] = self.conf["bz_api_key"]
```

Comment:
we might want to raise an exception here too, users will update their config to use the same API key if they want to test things, but at least it will be an explicit choice

Reply:
Done in 6d0e588ce79062692c774abbe684b449666586c4